src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborReaderTests.Integer.cs

```diff
@@ -114,6 +114,31 @@ public static void ReadUInt64_SingleValue_HappyPath(ulong expectedResult, string
             Assert.Equal(CborReaderState.Finished, reader.PeekState());
         }
 
+        [Theory]
+        [InlineData(0, "00")]
+        [InlineData(1, "01")]
+        [InlineData(10, "0a")]
+        [InlineData(23, "17")]
+        [InlineData(24, "1818")]
+        [InlineData(25, "1819")]
+        [InlineData(100, "1864")]
+        [InlineData(1000, "1903e8")]
+        [InlineData(1000000, "1a000f4240")]
+        [InlineData(byte.MaxValue, "18ff")]
+        [InlineData(byte.MaxValue + 1, "190100")]
+        [InlineData(ushort.MaxValue, "19ffff")]
+        [InlineData(ushort.MaxValue + 1, "1a00010000")]
+        [InlineData(int.MaxValue, "1a7fffffff")]
+        [InlineData(uint.MaxValue, "1affffffff")]
+        public static void ReadUInt32_SingleValue_HappyPath(uint expectedResult, string hexEncoding)
```

Comment:
Please add the overflow and bad input variant tests.

Reply:
I have the changes locally, but waiting on current CI to finish before I push them.

Reply:
Which seems might take a while :-/